src/index.js

```diff
@@ -0,0 +1,45 @@
+import './style.css';
+
+const listsContainer = document.querySelector('#tasks');
+const newListForm = document.querySelector('[data-new-list-form]');
+const newListInput = document.querySelector('[data-new-list-input]');
+
+const lists = [];
+
```

Comment:
- [ ] According to requirements, please create an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]

Reply:
well resolved! ✔️